src/modules/0.0.22/aws_ecr/rpcs/build.ts

```diff
@@ -0,0 +1,167 @@
+import { BatchGetBuildsCommandInput, Build, CodeBuild, CreateProjectCommandInput, DeleteSourceCredentialsCommandInput, ImportSourceCredentialsInput, Project, StartBuildInput } from "@aws-sdk/client-codebuild";
+import { createWaiter, WaiterState } from "@aws-sdk/util-waiter";
+import { AWS, crudBuilder2, crudBuilderFormat } from "../../../../services/aws_macros";
+import { Context, RpcBase, RpcResponseObject } from "../../../interfaces";
+import { BuildStatus } from "../../aws_codebuild/entity";
+import { awsIamModule } from "../../aws_iam";
+import { IamRole } from "../../aws_iam/entity";
+import { Repository, RepositoryImage } from "../entity";
+import { AwsEcrModule } from "../index";
+
```

Comment:
nit: in other places we use 2 spaces instead of 4

Reply:
indeed, running the `yarn format` script should fix it

Reply:
Yes, I've switched to VSCode and hadn't configured `prettier` yet.